Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: cancel during join or host

### Description (*)

This PR adds a cancel button to allow cancellation of joining or hosting attempts.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): #275 
### Manual testing scenarios

Those are easiest to test by using Unity Relay with Clumsy active, with a high simulated latency for receiving packets. Otherwise it happens quickly and is hard to cancel before it does. It can also be tested easily when trying to join via direct IP using the wrong IP.
1. Start BossRoom and attempt to host or join a game
2. Cancel hosting or joining
3. See that you are back at the main menu and can now try again
### Questions or comments

### Contribution checklist
